kernelci/db/kernelci_api.py

```diff
@@ -103,6 +103,16 @@ def get_root_node(self, node_id):
         resp = self._get('/'.join(['get_root_node', node_id]))
         return json.loads(resp.text)
 
+    def get_completed_event(self, node_id, hours=0, minutes=0, seconds=0):
+        """Get an event when all child nodes are complete of a given node"""
+        resp = self._get(f'trigger_completed_event/{node_id}?wait_time_hours=\
+{hours}&wait_time_minutes={minutes}&wait_time_seconds={seconds}')
```

Comment:
When using query parameters we should consider using params kyeword argument for requests.get(), which is used by _get method.
https://requests.readthedocs.io/en/latest/api/#requests.get